core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -119,15 +119,18 @@ angular.module('oppia').factory('QuestionObjectFactory', [
         }
       });
       if (pendingMisconceptionNamesToTag.length > 0) {
-        var returnString = 'Click on (or create) an answer ' +
-          'that is neither marked correct nor is a default answer (marked ' +
-          'above as [All other answers]) and tag the following misconceptions' +
-          ' to that answer group:';
+        var returnString = 'Remaining misconceptions that need to be ' +
```

Comment:
Add paren after '=' and break after that.

Reply:
Done.